airflow-core/src/airflow/api_fastapi/core_api/routes/public/dags.py

```diff
@@ -113,9 +115,19 @@ def get_dags(
     ],
     readable_dags_filter: ReadableDagsFilterDep,
     session: SessionDep,
+    user: GetUserDep,
+    favorites: Annotated[bool | None, Query()] = None,
```

Comment:
```suggestion
    is_favorite: Annotated[bool | None, Query()] = None,
```

I think we should create a reusable param out of this. Just to not clutter the `route` file, and also because it will be reusable.

Basically create a parameter, you can take a look at `common/parameters.py`. 
This will be more or less the `to_orm` method. Also I should be able to pass `False` to retrieve non favorited dag only. (if a value is defined for the filter, do the join and the filter, if it's None, don't do anything)
```python
            .join(DagFavorite, DagModel.dag_id == DagFavorite.dag_id)
            .where(DagFavorite.user_id == user_id)
```

Reply:
Done.